parl/env/compat_wrappers.py

```diff
@@ -24,6 +24,8 @@
 V_STEP_CHANGED = '0.25.0'
 V_NPRANDOM_CHANGED = '0.21.0'
 
+__all__ = ['CompatWrapper']
```

Comment:
please include all global variable and `get_gym_version` , will be used in https://github.com/PaddlePaddle/PARL/blob/develop/parl/env/atari_wrappers.py#L24

Reply:
done